src/app/lib/presentation/dad_jokes/favorite_dad_jokes.dart

```diff
@@ -27,15 +27,17 @@ final class FavoriteDadJokesPage extends ConsumerWidget {
       ),
       body: dadJokesAsyncValue.when(
         data: (dadJokes) {
-          return ListView.builder(
-            itemCount: dadJokes.length,
-            itemBuilder: (context, index) {
-              final dadJoke = dadJokes[index];
-              return DadJokeListItem(
-                key: Key(dadJoke.id),
-                dadJoke: dadJoke,
-              );
-            },
+          return Container(
```

Comment:
Isn't there a parameter key in the ListView builder ctor? That's a shame you need to wrap it into another widget just for tests

Reply:
there is a key parameter but for some unknow reason it doesn't enable us to find it's descendants with the finder